_maps/configs/komodo.json

```diff
@@ -0,0 +1,43 @@
+{
+		"prefix": "SSV",
+	"namelists": ["GENERAL", "NATURAL_AGGRESSIVE", "BEASTS", "WEAPONS"],
+	"map_name": "Komodo-class Warship",
+	"map_short_name": "Komodo-class",
+	"map_path": "_maps/shuttles/shiptest/gorlex_Komodo.dmm",
+	"map_id": "gorlex_Komodo",
+	"job_slots": {
+		"Captain": {
+			"outfit": "/datum/outfit/job/captain/syndicate/gorlex",
+			"officer": true,
+			"slots": 1
+		},
+		"Sergeant": {
+			"outfit": "/datum/outfit/syndicate_empty/sbc/assault/operative",
+			"officer": true,
+			"slots": 1
+		},
+		"Medic":{
+			"outfit": "/datum/outfit/job/doctor/syndicate_komodo",
+			"slots": 1
+		},
+		"Mechanic": {
+			"outfit": "/datum/outfit/job/engineer/syndicate/gorlex",
+			"slots": 1
+		},
+		"Trooper":{
+			"outfit": "/datum/outfit/syndicate/assult_komodo",
```

Comment:
change this path to /datum/outfit/syndicate_empty/assult_komodo, and also correct that to "assault_komodo" to fix the typo.

Reply:
this was fixed